docs/notebooks/passband-demo.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>Should we say where does it cache files by the default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tdastro/pull/382/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Added